set.mm

```diff
@@ -67281,22 +67283,21 @@ associative structure (such as a group).  (Contributed by NM,
 
   ${
     $d x y $.
-    $( The class of all ordinal numbers is ordinal.  Proposition 7.12 of
-       [TakeutiZaring] p. 38, but without using the Axiom of Regularity.
-       (Contributed by NM, 17-May-1994.) $)
-    ordon $p |- Ord On $=
-      ( vx vy con0 word wtr cep wwe tron wfr cv wbr weq wral onfr wcel eloni wa
-      w3o epel mpbir2an ordtri3or 3orbi123i sylibr syl2an rgen2a dfwe2 df-ord
-      biid ) CDCECFGZHUICFIAJZBJZFKZABLZUKUJFKZRZBCMACMNUOABCUJCOUJDZUKDZUOUKCO
-      UJPUKPUPUQQUJUKOZUMUKUJOZRUOUJUKUAULURUMUMUNUSBUJSUMUHAUKSUBUCUDUEABCFUFT
-      CUGT $.
-  $}
-
-  $( The epsilon relation well-orders the class of ordinal numbers.
-     Proposition 4.8(g) of [Mendelson] p. 244.  (Contributed by NM,
-     1-Nov-2003.) $)
-  epweon $p |- _E We On $=
-    ( con0 word cep wwe ordon ordwe ax-mp ) ABACDEAFG $.
+    $( The membership relation well-orders the class of ordinal numbers.
```

Comment:
Is it really wise to replace "epsilon relation" by "membership relation" if `_E` is involved? Both terms are synonymous, and "membership relation" is also used if ony `e.` is involved.

Reply:
To me, it is. The term "membership relation" is self-descriptive, while "epsilon relation" is slightly obscure and outdated.
I modified a while ago the comment of https://us.metamath.org/mpeuni/df-eprel.html to mention both terms with a short explanation for the use of the term "epsilon relation" (the use of the term "membership relation", on the other hand, does not require any explanation since, again, it is self-descriptive). To me, that comment could be the sole place where the term "epsilon relation" appears. I also modified around the same time the comments of epelg/epeli/epel to use the precise terms "membership relation" and "membership predicate".

Reply:
I like "membership relation" and "membership predicate".